frontend/src/pages/MapViewPage.js

```diff
@@ -54,13 +54,13 @@ class MapViewPage extends Component {
       <div>
         <Script url = {this.url} onLoad = {this.handleScriptLoad}/>
         <TopNavbar history={this.props.history} loggedIn={this.state.loggedIn} plus_code={this.state.plus_code}/>
-	  <Toast style={{backgroundColor: "white", position: "absolute", zIndex: 2, border: 0, borderRadius: "1rem", padding: 0, minWidth: "25rem", float: "right", margin: "1rem"}}>
-	    <Toast.Body>
-      	      <Card style={{border: 0}}>
-		<Form>
-		  <Form.Control style={{border: 0, focusOutline: "none"}} id = "autocomplete" placeholder = "Enter university"/>
-		</Form>
-      	      </Card>
+    <Toast style={{backgroundColor: "white", position: "absolute", zIndex: 2, border: 0, borderRadius: "1rem", padding: 0, minWidth: "25rem", float: "right", margin: "1rem"}}>
```

Comment:
The indentation here doesn't look right on github/when I click on the file on github.